Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -103,14 +106,17 @@ During this stage, the release manager has the following tasks:
 	* Remove from reviewers list any engineer that has been added by the PullAssigner but haven't contributed to the release branch.
 	* Set the _Merge_ label once all the required reviewers have approved it.
 1. Update the [release calendar](#release-calendar)
+	* Tip: You can use the following command in your Terminal to list the tickets and authors who participated in the Release to ask them the time they took for each ticket  
+          `git log --format="%<(25)%an | %s" develop..origin/release/babylon/4.15.0 | grep -vE "^(Steve|iOS Bot) *\|" | sort`
```

Comment:
🎉 

I'd maybe enhance it with `git fetch` in the beginning and reference origin develop (i.e. `origin/develop..origin/release/babylon/4.15.0`), just to make sure everything is up to date when running this command.

Reply:
Good point. Done 👍